pkg/diagnostics/cluster/registry.go

```diff
@@ -108,10 +108,6 @@ ownership/permissions.
 For volume permission problems please consult the Persistent Storage section
 of the Administrator's Guide.
 
-In the case of SELinux this may be resolved on the node by running:
```

Comment:
I _think_ this should be okay to remove, unless for some reason we are expecting people having trouble with hostpath PVs (which they should never be using anyway) to look here for a solution